jsapp/js/components/formLanding/formLanding.js

```diff
@@ -233,23 +234,26 @@ class FormLanding extends React.Component {
                 return (
                   <bem.FormView__group m='items' key={n} className={n >= this.state.DVCOUNT_LIMIT ? 'hidden' : ''}>
                     <bem.FormView__label m='version'>
-                      {`v${dvcount - n}`}
+                      <span>{`v${dvcount - n}`}</span>
                       {item.uid === this.state.deployed_version_id && this.state.deployment__active && (
-                        <bem.FormView__cell m='deployed'>{t('Deployed')}</bem.FormView__cell>
+                        <AssetStatusBadge deploymentStatus={this.state.deployment_status} />
                       )}
                     </bem.FormView__label>
                     <bem.FormView__label m='date'>{formatTime(item.date_deployed)}</bem.FormView__label>
                     {isLoggedIn && (
-                      <Button
-                        type='text'
-                        size='m'
-                        onClick={() => {
-                          this.saveCloneAs(item.uid)
-                        }}
-                        startIcon='duplicate'
-                        tooltip={t('Clone this version as a new project')}
-                        tooltipPosition='right'
-                      />
+                      <bem.FormView__label>
+                        <Button
```

Comment:
We already touch the code here, so let's swich to our new button `jsapp/js/components/common/ButtonNew.tsx` :)

Reply:
I'm actually a bit confused here. On `ButtonNew` it says to not use this for an icon with no text, and use ActionIcon instead. `ActionIcon` doesn't have a tooltip so just using that would be a regression. I think this:
```typescript
<Tooltip label={t('Clone this version as a new project')}>
  <ActionIcon
    size='md'
    variant='transparent'
    onClick={...}
>
  <Icon name='duplicate' size='l' />
```

Is so complicated for a "button and a tooltip", which `ButtonNew` would solve (but warns to not use for this case). Should I extend our `ActionIcon.tsx` to include tooltip?

Reply:
Hmm… I thought this would be simpler to do… 😓 If you have time for adding tooltip to `ActionIcon`, let's do it. But no pressure, given the situation, it's fine to leave old button too.